### ISSUE TITLE
Unsupported Cipher

### ISSUE THREAD

**Issue by sysblob** [alexjustesen/speedtest-tracker#1733]
**Describe the bug**
Unsupported cipher or incorrect key length. Supported ciphers are: aes-128-cbc, aes-256-cbc, aes-128-gcm, aes-256-gcm.

**To Reproduce**
It appears ookla updated their cipher requirements and the base64 key you're handing out on your dev website no longer works to run this application.

**Comment by svenvg93** [comment]
Do you have the full key set in env value? 
Make sure it contains the `base64` at the start and the `=` at the end.

**Comment by sysblob** [comment]
> Do you have the full key set in env value? Make sure it contains the `base64` at the start and the `=` at the end.

I did -- I'm honestly not sure what caused my issue but it seems resolved now. One thing I did notice is the docker compose in your github readme differs from the docker compose you supply in your documentation and on linuxserver.io. When I switched over to the documentation version it began working. Not sure if coincidental but one thing it removes is APP_URL.
